custom-connectors/Datelist/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "API Key",
+          "tooltip": "You can find it here: https://datelist.io/users/edit",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#818BE1",
+    "capabilities": [],
+    "publisher": "Alexis Clarembeau"
```

Comment:
We'd recommend that the `publisher` and `stackOwner` properties do not contain PII. It's better for it to be representative of the company as a whole.